qa/restart-upgrade/src/test/resources/processor/PipelineConfiguration.json

```diff
@@ -0,0 +1,13 @@
+{
+  "description": "text embedding pipeline for hybrid",
+  "processors": [
+    {
+      "text_embedding": {
+        "model_id": "%s",
+        "field_map": {
+          "passage_text": "passage_embedding"
+        }
```

Comment:
in the follow up let's make sure that we are adding the tests for array fields and map fields in fieldMap